code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,322 @@
+	colour = "dark blue"
+
+/obj/item/slimecross/destabilized/darkblue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.apply_status_effect(/datum/status_effect/freon/destabilized)
+
+/obj/item/slimecross/destabilized/silver
+	colour = "silver"
+
+/obj/item/slimecross/destabilized/silver/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.nutrition = NUTRITION_LEVEL_FAT + 1
+		to_chat(L, "<span class='warning'>You suddenly feel gorged!</span>")
+
+/obj/item/slimecross/destabilized/bluespace
+	colour = "bluespace"
+
+/obj/item/slimecross/destabilized/bluespace/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		do_teleport(A, get_turf(thrower), 1)
+	if(isobj(A))
+		var/obj/O = A
+		if(!O.anchored)
+			do_teleport(O, get_turf(thrower), 1)
+
+/obj/item/slimecross/destabilized/sepia
+	colour = "sepia"
+
+/obj/item/slimecross/destabilized/sepia/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/datum/proximity_monitor/advanced/timestop/TS = new()
+		TS.host = A
+		TS.freeze_atom(A)
+		to_chat(A, "<span class='warning'>You suddenly feel every molecule in your body grind to a halt!</span>")
+		QDEL_IN(TS, 70)
+	else
+		new /obj/effect/timestop(get_turf(A), null, 50)
+
+/obj/item/slimecross/destabilized/cerulean
+	colour = "cerulean"
+
+/obj/item/slimecross/destabilized/cerulean/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(ismovableatom(A))
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), A, thrower)
+	else
+		new /mob/living/simple_animal/hostile/clone(get_turf(A), thrower, thrower)
+
+/obj/item/slimecross/destabilized/pyrite
+	colour = "pyrite"
+
+/obj/item/slimecross/destabilized/pyrite/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/turf/T = get_turf(A)
+	T.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+	for(var/atom/atoms in T)
+		atoms.add_atom_colour("#[rand_hex_color()]", WASHABLE_COLOUR_PRIORITY)
+
+/obj/item/slimecross/destabilized/red
+	colour = "red"
+
+/obj/item/slimecross/destabilized/red/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(iscarbon(A))
+		var/mob/living/carbon/C = A
+		C.apply_status_effect(/datum/status_effect/heavyBleeding)
+	else
+		new /obj/effect/decal/cleanable/blood(get_turf(A))
+
+/obj/item/slimecross/destabilized/green
+	colour = "green"
+
+/obj/item/slimecross/destabilized/green/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A) && isliving(thrower))
+		var/mob/living/LTarget = A
+		var/mob/living/LThrower = thrower
+		var/damageToDeal
+		if(iscarbon(LThrower))
+			var/mob/living/carbon/CThrower = LThrower
+			var/damageType = findBiggestLoss(CThrower)
+			damageToDeal = min(CThrower.get_damage_amount(damageType), 25)
+			CThrower.apply_damage_type(-25, damageType)
+			if(iscarbon(LTarget))
+				var/mob/living/carbon/CTarget = LTarget
+				CTarget.apply_damage_type(damageToDeal, damageType)
+				return
+			LTarget.adjustBruteLoss(damageToDeal)
```

Comment:
why doesn't this apply damage of the healed type?

Reply:
It does, unless the target mob is non-carbon, in which case it makes sense just to do brute damage, as that is what all other damage types get converted to when applied to simplemobs.